src/components/graph/nodes/network-modification-node.js

```diff
@@ -11,37 +11,70 @@ import makeStyles from '@mui/styles/makeStyles';
 import { useSelector } from 'react-redux';
 import CircularProgress from '@mui/material/CircularProgress';
 import VisibilityIcon from '@mui/icons-material/Visibility';
-import BuildIcon from '@mui/icons-material/Build';
+import VisibilityOffIcon from '@mui/icons-material/VisibilityOff';
 import LockIcon from '@mui/icons-material/Lock';
-import clsx from 'clsx';
+import Tooltip from '@mui/material/Tooltip';
 
 const useStyles = makeStyles((theme) => ({
-    networkModificationSelected: {
+    networkModificationVisualized: {
+        position: 'relative',
         variant: 'contained',
         background: theme.palette.primary.main,
         textTransform: 'none',
         color: theme.palette.primary.contrastText,
         '&:hover': {
             background: theme.palette.primary.main,
         },
+        overflow: 'hidden',
     },
     networkModification: {
-        variant: 'outlined',
-        background: theme.palette.primary.light,
+        background: '#c5c8d1',
```

Comment:
In dark mode this works fine, but in light mode, the display is really hard to see.
Maybe use the theme.palette to help with contrast ?

`background: theme.palette.text.secondary,`
works fine here.

Reply:
Agreed `background: theme.palette.text.secondary,` works well in both themes for the nodes background